digestiflow_demux/bases_mask.py

```diff
@@ -0,0 +1,96 @@
+class BaseMaskConfigException(Exception):
+    """Raise if base mask for demux_reads is malformed"""
+
+    pass
+
+
+def split_bases_mask(bases_mask):
+    """Parse a picard style bases mask
+    and return a list of tuples
+    e.g. for paired-end dual-indexed runs
+    [('T', 100), ('B', 8), ('B', 8), ('T', 100)]"""
+    splitat = []
+    for i, c in enumerate(bases_mask):
+        if c.isalpha():
+            splitat.append(i)
```

Comment:
Could also be done as here https://github.com/bihealth/digestiflow-web/blob/1593d43d2535d0e2115b19cd1a20a0aea4343fa4/flowcells/models.py#L331

Reply:
If you allow, I would keep it as is without using regular expressions.

Reply:
OK